DirectProgramming/C++SYCL_FPGA/Tutorials/Features/ip_authoring_interfaces/component_interfaces_comparison/pipes/Readme.md

```diff
@@ -0,0 +1,175 @@
+# Streaming Data Interfaces (Pipes)
+This implementation uses a register-mapped invocation interface, and demonstrates how to use pipes to configure a streaming data interface.
+
+![](../assets/agent_n_pipe.svg)
+
+## Invocation Interface
+By default, an un-decorated oneAPI kernel will have all its control signals and arguments mapped into the IP component's control/status register (CSR).
+
+## Data Interface - Streaming (Pipes)
+Pipes are a first-in first-out (FIFO) buffer construct that provide links between elements of a design. In this design, interfaces `InputPipeA`, `InputPipeB`, `OutputPipeC` are implemented as  pipes.
+
+### Declaring a Pipe
+Each pipe is a declaration of the templated `pipe`` class:
+```cpp
+sycl::ext::intel::experimental::pipe<id, type, min_capacity, properties>;
+```
+Detailed explanation for each parameters and a list of properties can be found in this dedicated [Streaming Data Interfaces](https://github.com/oneapi-src/oneAPI-samples/tree/master/DirectProgramming/C%2B%2BSYCL_FPGA/Tutorials/Features/ip_authoring_interfaces/streaming_data_interfaces) code sample.
```

Comment:
Dead link.

Reply:
Yes, this link is pending this PR: https://github.com/oneapi-src/oneAPI-samples/pull/1863

Reply:
You can replace this with an absolute link to where the sample will be locate: `DirectProgramming/C++SYCL_FPGA/Tutorials/Features/ip_authoring_interfaces/streaming_data_interfaces`

Github will deduce the appropriate branch where this belongs depending on the branch this sample is in.